### DISCUSSION TITLE
How to refresh buffer?

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#3441] [Q&A] [answered]
Is there a way to refresh the current working buffer? Similar to `:e` or `:redraw` in Neovim, in case an external program, or similar, modifies the file and helix doesn't update its contents. Currently I have to close the buffer and open again. I admit it's not much of hassle but it would be convenient not to close.

**Comment by the-mikedavis** [comment, marked as answer]
There's a `:reload` command which reloads the file from disk
